lib/inertia_rails/controller.rb

```diff
@@ -20,17 +20,31 @@ def inertia_share(**args, &block)
       end
     end
 
+    private
```

Comment:
Somewhat unrelated to the specific change, but I _think_ we'd always want these methods to be private. Happy to revert if there are any concerns with this.

Reply:
Wow, TIL that I did not understand method visibility in ruby as well as I thought I did; very different than PHP. Cool stuff!

Reply:
I'm a bit late here, but `redirect_to` needs to be `public`, which is the Rails default. Changing it to `private` causes issues when using Devise, so I reverted it in #60.

Reply:
Sorry for the issue here!